packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -261,6 +262,7 @@ export class CalloutContentBase extends React.Component<ICalloutProps, ICalloutS
             onScroll={onScroll}
             shouldRestoreFocus={shouldRestoreFocus}
             style={overflowStyle}
+            {...popupProps}
             onMouseDown={this._mouseDownOnPopup}
             onMouseUp={this._mouseUpOnPopup}
```

Comment:
nit: Any particular reason this doesn't let popupProps override the onMouseDown/onMouseUp handlers? It looks like the original PR #24212 did allow it to override them.

```suggestion
            onMouseDown={this._mouseDownOnPopup}
            onMouseUp={this._mouseUpOnPopup}
            {...popupProps}
```